src/Sylius/Behat/Context/Setup/CustomerContext.php

```diff
@@ -182,6 +182,7 @@ public function iHaveAnAddressInAddressBook(ShopUserInterface $user, AddressInte
     {
         /** @var CustomerInterface $customer */
         $customer = $user->getCustomer();
+        $this->sharedStorage->set('address', $address);
```

Comment:
Will be redundant after https://github.com/Sylius/Sylius/pull/6526/files#r85093765